test/functional/page_objects/discover_page.ts

```diff
@@ -129,6 +129,12 @@ export function DiscoverPageProvider({ getService, getPageObjects }: FtrProvider
       await testSubjects.click('discoverOpenButton');
     }
 
+    public async clickResetSavedSearchButton() {
+      await testSubjects.moveMouseTo('resetSavedSearch');
```

Comment:
nit: This line seems to be unnecessary

Reply:
I just did it the way other methods work in this class, dear @LeeDr, is it redundant?

Reply:
I *thought* `testSubjects.click()` which calls `find.clickByCssSelector()` always moved the mouse to the element before clicking it.  But it turns out that `moveMouseTo` was commented-out around a year ago.

But in general, you should not have to move the mouse to an element before clicking it unless;
  * It has a hover event that causes an element to appear or be enabled
  * It's off the screen and moving the mouse to it causes it to be scrolled into view

I don't think it's necessary in this case.  But it also doesn't hurt.  At this point where you've already run flaky test runner, I would probably leave it.

Reply:
ok, thx for information, I'll merge 👍